client/src/components/FeedbackForm.js

```diff
@@ -2,22 +2,38 @@
 import React, { useState } from "react";
 import "./feedbackForm.css";
 
-const FeedbackForm = ({ student }) => {
-	console.log("student",student);
-	const [feedbackMessage, setFeedbackMessage] = useState("");
+const FeedbackForm = () => {
+	const [feedbackMessage, setFeedbackMessage] = useState({
+		feedback: "",
+		name: "",
+	});
 	const [givenFeedback, setGivenFeedback] = useState([]);
 	const [feedbackList, setFeedbackList] = useState([]);
 
 	const handleSubmit = (e) => {
 		e.preventDefault();
 		setGivenFeedback(feedbackMessage);
```

Comment:
@leealex88 in here are both "feedbackMessage" and "givenFeedback" states equal? 
Also, why do we need to keep track of givenFeedback? Can we just post it to the database and then fetch it to populate our "previousFeedback" which could be a separate component?

Reply:
That givenfeedback was there before and it is redundant. No problem to delete it.

Reply:
yes, exactly it was there because it was needed to display it - now we can delete it.